translations

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env node
+
+/**
+ * A translations too that can be used to keep a translation file(s)
+ * up to date with one source language file.
+ *
+ * To track new languages add a desired language to the TARGET_LANGS array.
+ * By default the translations are matched agains English translations
+ * but that can be changed by modifying the SOURCE_LANG.
+ */
+
+const inquirer = require('inquirer');
+const difference = require('lodash/difference');
+const fs = require('fs');
+const chalk = require('chalk');
+
+const PATH = './src/translations/';
+
+const SOURCE_LANG = { name: 'English', value: 'en' };
+const TARGET_LANGS = [
+  { name: 'Spanish', value: 'es' },
+  { name: 'German', value: 'de' },
+  { name: 'French', value: 'fr' },
+];
+
+class BreakSignal {}
+
+const run = () => {
+  let sourceLang;
+  let targetLang;
+  let source;
+  let target;
+  let sourceKeys;
+  let targetKeys;
+  let key;
+
+  selectLanguage()
+    .then(answers => {
+      sourceLang = SOURCE_LANG.value;
+      targetLang = answers.lang;
+      source = require(filePath(sourceLang));
+      target = require(filePath(targetLang));
```

Comment:
It's probably better to not use `require` to future-proof this since _"Require is synchronous function and is called only once, which means the calls receive a cached result. If the file is updated you can't re-read it using this method"_

https://stackabuse.com/reading-and-writing-json-files-with-node-js/